### ISSUE TITLE
A naming error

### ISSUE THREAD

**Issue by laisimiao** [xiaozai/DeT#3]
Following sequence_list tell us there is a seq called `lock_wild` in depthtrack test set, but actully it's in training set link you provide and called `lock01_wild`. At the same time, there is a seq called `lock02_indoor` in test set link you provide.
https://github.com/xiaozai/DeT/blob/8aa5e2eae91e0f3f88290b4133aadf09aebb34b6/pytracking/evaluation/depthtrackdataset.py#L114

Please note this.

**Comment by xiaozai** [comment]
Thanks! I fixed it. I forget that I moved that sequence into a training set and forget to update the GitHub.
